### DISCUSSION TITLE
MongoDB with vendure

### DISCUSSION THREAD

**Discussion by zohra-alt** [vendurehq/vendure#812] [General]
Hello!

Is it possible to use vendure with mongodb as a database?

**Comment by michaelbromley** [comment]
Hi,

No, MongoDB is not supported. The reason is that Vendure uses TypeORM, which only partially supports MongoDB. Furthermore I would suggest that a document database is not well-suited to the highly relational nature of the e-commerce data model.

**Reply by zohra-alt** [reply]
Okay, I see. Thanks for the response and the advice.
